src/components/common/modal/Backdrop.tsx

```
@@ -0,0 +1,22 @@
+
+export const Backdrop = ({ className, ...props }: BackdropProps) => {
+  const { onOpenChange } = useModalContext();
+  const handleClick = () => {
+    onOpenChange(false);
+  };
+  return (
+    <div
+      className={`fixed inset-0 bg-black/50 backdrop-blur-sm z-50 ${className}`}
+      onClick={handleClick}
+      {...props}
+    ></div>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)